CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ option(OPUS_INSTALL_PKG_CONFIG_MODULE "Install PkgConfig module" ON)
 option(OPUS_INSTALL_CMAKE_CONFIG_MODULE "Install CMake package config module"
        ON)
 
+if(APPLE)
+  option(BUILD_FRAMEWORK "Build Framework bundle for Apple systems" OFF)
```

Comment:
add the option below were all the other options are specified so it is printed out into the help